package-lock.json

```diff
@@ -3328,6 +3366,24 @@
         "node": ">= 0.10.0"
       }
     },
+    "node_modules/express-jwt": {
+      "version": "8.4.0",
+      "resolved": "https://registry.npmjs.org/express-jwt/-/express-jwt-8.4.0.tgz",
+      "integrity": "sha512-AvIkHI6+wuwHQTgfnl4kEPWeMKo5yw4FnXJJK+jf/PRWAflmuJKTs06ENRNJ6sCQceIUVqAi/fy8Nav8alnv0w==",
+      "dependencies": {
+        "@types/jsonwebtoken": "^9",
```

Comment:
This could be the culprit, new non-dev dependency `express-jwt` declares dependency on `@types/jsonwebtoken` without saying that one should be dev dependency.